deployment/network-operator/README.md

```diff
@@ -0,0 +1,163 @@
+# Nvidia Mellanox Network Operator Helm Chart
+
+Nvidia Mellanox Network Operator Helm Chart provides an easy way to install, configure and manage
+the lifecycle of Nvidia Mellanox network operator.
+
+## Nvidia Mellanox Network Operator
+Nvidia Mellanox Network Operator leverages [Kubernetes CRDs](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
+and [Operator SDK](https://github.com/operator-framework/operator-sdk) to manage Networking related Components in order to enable Fast networking, 
+RDMA and GPUDirect for workloads in a Kubernetes cluster.
+
+The Goal of Network Operator is to manage _all_ networking related components to enable execution of
+RDMA and GPUDirect RDMA workloads in a kubernetes cluster including:
```

Comment:
maybe we should mention that it is working together with the GPU operator

Reply:
yes i will mention it